pylint_tweaks

```diff
@@ -0,0 +1,5 @@
+[MASTER]
+load-plugins = edx_lint.pylint
+
+[MESSAGES CONTROL]
+disable = django-not-available
```

Comment:
Do you actually need this `disable` setting with pylint_django having been removed from `load-plugins` above?  I didn't need it in the edx-sphinx-theme repo, which also doesn't use Django